packages/ag-charts-community/src/scene/scene.ts

```diff
@@ -308,6 +308,7 @@ export class Scene {
 
         const renderCtx: RenderContext = {
             ctx,
+            devicePixelRatio: this.overrideDevicePixelRatio ?? 1,
```

Comment:
Use `this.canvas.pixelRatio` as otherwise the fallback of `1` is incorrect (and semantically makes more sense since it's the pixel ration reported for the canvas)?